app/ui/omni-box.js

```diff
@@ -35,6 +35,10 @@ class OmniBox extends HTMLElement {
       this.input.select()
     })
 
+    this.input.addEventListener('blur',  () => {
+      this.input.blur()
```

Comment:
Does removing this part change anything? I would imagine that `'blur` only gets emitted when `this.input` is already blurred.

Reply:
Yea I know it's weird! The document.activeElement does not get updated upon clicking on the webContents, adding an explicit blur like this fixes it.

I'm guessing it could be something related to electron sharing focus between processes?